packages/neon_lints/lib/flutter.yaml

```diff
@@ -1,9 +1,6 @@
-include: package:neon_lints/dart.yaml
+include: package:neon_lints/src/base.yaml
+
 linter:
   rules:
     public_member_api_docs: false
     library_private_types_in_public_api: false
-    avoid_print: true
```

Comment:
I think we should keep this as one should use `debugPrint` instead.

Reply:
sure but isn't this already required by the depending on flutter_package.yaml?

Reply:
Currently it depends on base and not flutter_package